### PR TITLE
DLU release

### PR DESCRIPTION
**Pull request by ArkShocer**
- Changed Status
- Added Github link
- Updated "How do I get a server emulator?" paragraph

### CODE REVIEW

**Comment by Jettford** [comment]
@luxaritas

**Review by Xiphoseer — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -154,8 +154,9 @@ <h3 id="how-to-client">How do I get a game client? Can I use my live client? Can
             </ul>
             <h3 id="how-to-server">How do I get a server emulator?</h3>
             <p>
-                DLU has announced they will be releasing their server emulator, most likely before the end of the year.
-                We will update this information at that point. Until then, you may be interested in setting up Uchu:
+                DLU has <a href="https://github.com/DarkflameUniverse/DarkflameServer">released</a> their server emulator, you can finally play what should be a feature-complete version of LEGO Universe!
+                For DLU setup questions please use the  <a href="https://discord.gg/jhTwGqYSKk" target="_blank"><i class="fab fa-discord"></i> LU Community Hub</a>.
```
Comment (luxaritas) [inline index.html:158]:
```suggestion
                DLU has released <a href="https://github.com/DarkflameUniverse/DarkflameServer">their server emulator</a> which allows you to have a nearly feature-complete experience of LEGO Universe.
                For DLU setup questions, the best place to ask is the <a href="https://discord.gg/jhTwGqYSKk" target="_blank"><i class="fab fa-discord"></i>LU Community Hub</a>.
```

**Review by luxaritas — CHANGES_REQUESTED** [review]
Minor wording tweak I think would work better - otherwise thanks for getting this updated!

**Review by luxaritas — APPROVED** [review]
LGTM, thanks!

**Pull request merged**